rules/windows/privilege_escalation_unusual_parentchild_relationship.toml

```diff
@@ -11,39 +11,49 @@ activity on a system.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License"
 name = "Unusual Parent-Child Relationship"
+references = [
+    "https://github.com/sbousseaden/Slides/blob/master/Hunting MindMaps/PNG/Windows Processes TH.map.png",
+]
 risk_score = 47
 rule_id = "35df0dd8-092d-4a83-88c1-5151a804f31b"
 severity = "medium"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.parent.executable:* and
-(process.parent.name:autochk.exe and not process.name:(chkdsk.exe or doskey.exe or WerFault.exe) or
-   process.parent.name:smss.exe and not process.name:(autochk.exe or smss.exe or csrss.exe or wininit.exe or winlogon.exe or WerFault.exe) or
-   process.name:autochk.exe and not process.parent.name:smss.exe or
-   process.name:(fontdrvhost.exe or dwm.exe) and not process.parent.name:(wininit.exe or winlogon.exe) or
-   process.name:(consent.exe or RuntimeBroker.exe or TiWorker.exe) and not process.parent.name:svchost.exe or
-   process.name:wermgr.exe and not process.parent.name:(svchost.exe or TiWorker.exe) or
-   process.name:SearchIndexer.exe and not process.parent.name:services.exe or
-   process.name:SearchProtocolHost.exe and not process.parent.name:(SearchIndexer.exe or dllhost.exe) or
-   process.name:dllhost.exe and not process.parent.name:(services.exe or svchost.exe) or
-   process.name:smss.exe and not process.parent.name:(System or smss.exe) or
-   process.name:csrss.exe and not process.parent.name:(smss.exe or svchost.exe) or
-   process.name:wininit.exe and not process.parent.name:smss.exe or
-   process.name:winlogon.exe and not process.parent.name:smss.exe or
-   process.name:(lsass.exe or LsaIso.exe) and not process.parent.name:wininit.exe or
-   process.name:LogonUI.exe and not process.parent.name:(wininit.exe or winlogon.exe) or
-   process.name:services.exe and not process.parent.name:wininit.exe or
-   process.name:svchost.exe and not process.parent.name:(MsMpEng.exe or services.exe) or
-   process.name:spoolsv.exe and not process.parent.name:services.exe or
-   process.name:taskhost.exe and not process.parent.name:(services.exe or svchost.exe) or
-   process.name:taskhostw.exe and not process.parent.name:(services.exe or svchost.exe) or
-   process.name:userinit.exe and not process.parent.name:(dwm.exe or winlogon.exe))
+process where event.type in ("start", "process_started") and
```

Comment:
this query is crushing my recently built stack. is this going to be too resource intensive, or is it just me? @rw-access 
i have 3 ES nodes and it hasn't completed successfully for me, just hanging. has it completed successfully in your testing, or was it just a conversion for now? we may want to skip this conversion for now.

Reply:
it's possible that this contains too many insensitive matches. they are a little slower than case sensitive but it could be that they are all adding up.

best way to test would be to convert `:` to `in` and `==` and see how it goes

Reply:
is it possible if an EQL query is too resource intensive it times out after X amount of time instead of potentially freezing a cluster?

Reply:
looks like putting every OR block in between () speedup the query, need to try same on endpoint dev with new ":" insensitive (seems down):
 
![image](https://user-images.githubusercontent.com/64742097/101087268-4ac7f000-35b2-11eb-92c5-991f52c80bb6.png)

Reply:
tested and it works 

![image](https://user-images.githubusercontent.com/64742097/101099524-1578cd80-35c5-11eb-8b8f-48d36f31e706.png)

Reply:
it now completed very fast for me as well, thanks!